src/Microsoft.AspNetCore.Mvc.TagHelpers/Internal/FileVersionProvider.cs

```diff
@@ -111,7 +111,14 @@ public string AddFileVersionToPath(string path)
 
         private static string GetHashForFile(IFileInfo fileInfo)
         {
-            using (var sha256 = SHA256.Create())
+            HashAlgorithm sha256;
+#if NETSTANDARD1_6
+            sha256 = SHA256.Create();
+#else
+            sha256 = new SHA256Cng();
```

Comment:
@blowdart provided this pattern offline and was one of the ones to review the NuGet code you're referencing. I'm assuming there's some difference he could enlighten us on :smile: